From 8375d87ffd966de533deb53e344da75e0e509821 Mon Sep 17 00:00:00 2001
From: Mark Enriquez <enriquezmark36@gmail.com>
Date: Sat, 11 Apr 2020 11:12:46 +0800
Subject: [PATCH] ril: kanas: Use the scx15 version

We don't even use the modem_control daemon, even from the stock rom.

Signed-off-by: Mark Enriquez <enriquezmark36@gmail.com>
---
 ril/sc8830/Android.mk | 2 ++
 ril/scx15/Android.mk  | 4 +++-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/ril/sc8830/Android.mk b/ril/sc8830/Android.mk
index 25a94d6..378a09d 100644
--- a/ril/sc8830/Android.mk
+++ b/ril/sc8830/Android.mk
@@ -1,5 +1,7 @@
 LOCAL_PATH := $(call my-dir)
 
 ifeq ($(TARGET_BOARD_PLATFORM),sc8830)
+ifneq ($(SOC_SCX35),true)
 include $(call all-makefiles-under,$(LOCAL_PATH))
 endif
+endif
diff --git a/ril/scx15/Android.mk b/ril/scx15/Android.mk
index dd8624f..814de6d 100644
--- a/ril/scx15/Android.mk
+++ b/ril/scx15/Android.mk
@@ -1,5 +1,7 @@
 LOCAL_PATH := $(call my-dir)
 
-ifeq ($(TARGET_BOARD_PLATFORM),scx15)
+ifneq ($(filter $(TARGET_BOARD_PLATFORM), scx15) $(filter true, $(SOC_SCX35)),)
 include $(call all-makefiles-under,$(LOCAL_PATH))
 endif
+
+
-- 
2.25.0

